internal/service/service.go

```diff
@@ -55,6 +55,7 @@ type MqttService interface {
 
 type TreeService interface {
 	Service
+	ImportTree(ctx context.Context, row [][]string) error
```

Comment:
Only the object known for the domain should be used here, so please transfer the tree structure. The rows of the csv files should be mapped to this tree struct.

```suggestion
	ImportTree(ctx context.Context, trees []*domain.Tree) error
```